class1/02_palindromicAndPrimes.rb

```diff
@@ -0,0 +1,49 @@
+class SequenceUnion
+	attr_accessor:elementsToPrint
+	attr_accessor:arraySequence
+	def initialize
+		@elementsToPrint=200
+		@arraySequence=Array.new
+	end
+	def print_prime_palindromic_numbers
+		prime=1
+
+		for numPrimes in (1..elementsToPrint)
+			
+			if is_prime?(numPrimes)
+				if is_palindromic?(numPrimes)
+					arraySequence.push prime	
+				end
+			end
+		prime+=1		
+		end
+	end
+	def is_prime?(value)
+		isPrime=true
+		for divisor in (2..value-1)
+			if value%divisor==0
+				isPrime=false
+			end
+		end
+		return isPrime
+	end
+	def is_palindromic?(value)
+		reversePrimeNumber=Array.new
+		primeNumber=value.to_s.split(//)
+
+		if primeNumber.length<2
+			return true
+		end
+		while primeNumber.length>0
+				reversePrimeNumber.push((primeNumber.pop))	
```

Comment:
I think that is not necessary to have 2 pair of parenthesis 
reversePrimeNumber.push(primeNumber.pop)